src/components/Note/Note.tsx

```diff
@@ -49,7 +49,7 @@ export const Note = (props: NoteProps) => {
 
   useEffect(() => {
     if (isShared) {
-      if (!document.location.pathname.endsWith(props.noteId)) {
+      if (!document.location.pathname.endsWith(props.noteId + "/stack")) {
```

Comment:
took me weeks to find and fix this bug lol

Reply:
Is this missing in line 45 too?

Reply:
yes, I added it. thanks